kymatio/scattering2d/backend/tensorflow_backend.py

```diff
@@ -33,7 +33,7 @@ def __call__(self, x):
         if self.pre_pad:
             return x
         else:
-            paddings = [[0, 0]]*len(x.shape[:-2])
+            paddings = 2 * [[0, 0]]
```

Comment:
Actually, this was a bad idea: this is simply adding the necessary 0 for other dimensions. Gonna revert.

Reply:
Ah ok. With batch reshaping happening in the frontend, though, it should be a problem to assume everything has four dimensions (batch, channel, x, y).